react-native/ReactNativeFlipperExample/ios/ReactNativeFlipperExample.xcodeproj/project.pbxproj

```diff
@@ -836,6 +836,7 @@
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 i386";
```

Comment:
Why are we excluding this archs ?

Reply:
It's coming from 'react_native_post_install' script
the RN 0.64.x uses this function on Podfile but I think someone missed this